apps/api/v2/src/modules/atoms/services/event-types-atom.service.ts

```diff
@@ -59,19 +61,21 @@ export class EventTypesAtomService {
       ? await this.membershipsRepository.isUserOrganizationAdmin(user.id, organizationId)
       : false;
 
-    const eventType = await getEventTypeById({
-      currentOrganizationId: this.usersService.getUserMainOrgId(user),
-      eventTypeId,
-      userId: user.id,
-      prisma: this.dbRead.prisma as unknown as PrismaClient,
-      isUserOrganizationAdmin,
-      isTrpcCall: true,
-    });
-
-    if (!eventType) {
```

Comment:
I don't think this actually worked as getEventTypeById threw a TRPCError when the resource was not found, so !eventType was actually impossible.